website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md

```diff
@@ -0,0 +1,203 @@
+
+<Lightbox src="/img/guides/databricks-guides/databricks-connection-env-vars.png" title="Using environment variable syntax in connection configs" />
+
+When you create environments in dbt Cloud, you can assign environment variables to populate the connection information dynamically. Don’t forget to make sure the tokens you use in the credentials for those environments were generated from the associated workspace.
+
+<Lightbox src="/img/guides/databricks-guides/databricks-env-variables.png" title="Defining default environment variable values" />
+
+### Access Control
+
+For granting access to data consumers, use dbt’s [grants config](https://docs.getdbt.com/reference/resource-configs/grants) to apply permissions to database objects generated by dbt models. This lets you configure grants as a structured dictionary rather than writing all the SQL yourself and lets dbt take the most efficient path to apply those grants.
+
+As for permissions to run dbt and read non-consumer-facing data sources, the table below summarizes an access model. Effectively, all developers should get no more than read access on the prod catalog and write access in the dev catalog. When using dbt, schema creation is taken care of for you; unlike traditional data warehousing workflows, you do not need to manually create any Unity Catalog assets other than the top-level catalogs.
+
+The **prod** service principal should have “read” access to raw source data, and “write” access to the prod catalog. If you add a **test** catalog and associated dbt environment, you should create a  dedicated service principal. The test service principal should have *read* on raw source data, and *write* on the **test** catalog but no permissions on the prod or dev catalogs. A dedicated test environment should be used for [CI testing](https://www.getdbt.com/blog/adopting-ci-cd-with-dbt-cloud/) only.
+
+
+**Table-level grants:**
+
+|  | Source Data | Development catalog | Production catalog | Test catalog |
+| --- | --- | --- | --- | --- |
+| developers | select | select & modify | select or none | none |
+| production service principal | select | none | select & modify | none |
+| Test service principal | select | none | none | select & modify |
+
+
+**Schema-level grants:**
+
+|  | Source Data | Development catalog | Production catalog | Test catalog |
+| --- | --- | --- | --- | --- |
+| developers | use | use, create table & create view | use or none | none |
+| production service principal | use | none | use, create table & create view | none |
+| Test service principal | use | none | none | use, create table & create view |
+
+
+## Next steps
```

Comment:
I would love to ask @joellabes and other folks on the product docs team if we need these next steps. Most of these are already linking to our docs so perhaps we don't actually need them?

Reply:
I don't mind it! A link to the best practices guide makes sense